pkg/ifreload/ifreload.go

```diff
@@ -58,7 +59,8 @@ func (i *ifReload) ifreloadImage() string {
 func (i *ifReload) ensureNamespace(ns string) error {
 	_, err := i.cli.CoreV1().Namespaces().Create(&v1.Namespace{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: ns,
+			Name:        ns,
+			Annotations: map[string]string{"openshift.io/nodeselector": ""},
```

Comment:
There's a third place that this fix needs to be applied - please check `func (g *genevaLogging) ensureNamespace` too.

Reply:
I was naughty and copy/pasted code for ifreload and routefix from genevalogging.  Maybe you might help me by refactoring to consolidating ensureNamespace, apply{ConfigMap,DaemonSet}, etc. from those three packages into a single pkg/util/ensure ?